src/common/dao/artifact.go

```diff
@@ -26,6 +26,9 @@ import (
 func AddArtifact(af *models.Artifact) (int64, error) {
 	now := time.Now()
 	af.CreationTime = now
+	af.PushTime = now
+	af.PullTime = now
```

Comment:
The `PushTime` and `PullTime` should be set at upper level. Besides, the `PullTime` isn't the time when the artifact record is created in database.

Reply:
The assignment for PullTime was removed, but PushTime was kept for the time to create artifact is the most time to push the image to registry.